content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -23,12 +29,6 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
 RuntimeClass is a beta feature (as of v1.14) for selecting the container runtime configuration to
 use to run a pod's containers.
 
```

Comment:
nit: I don't think you need to repeat that this feature is beta in 1.14.
possible edit: RuntimeClass is a feature for selecting the container runtime configuration. The container runtime configuration is used to run a Pod's containers.